frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -138,6 +138,17 @@ object FrontendApp extends JSApp with Persistence with PrettyJson with Manipulat
   private[this] val graphView = new GraphView(showUnconnected, actorSelector.toggleActor, ActorStateAsNodeRenderer.render)
   private[this] val maxRetries = 10
 
+  private[this] var upstreamConnection: js.UndefOr[Upstream] = js.undefined
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [upstreamSend should have a smaller scope by using private[this]](https://www.codacy.com/app/lustefaniak/akka-viz/file/2183085965/issues/source?bid=3121282&fileBranchId=3164965#l143)